test/opentok-test.js

```diff
@@ -295,6 +295,15 @@ describe('when initialized with a proxy', function () {
   });
 });
 
+describe('when initialized with a timeout', function () {
+  beforeEach(function () {
+    this.opentok = new OpenTok(apiKey, apiSecret, { timeout: 100 });
```

Comment:
Can we add another test where if we don't pass a timeout?

Reply:
Sure thing!